style.css

```diff
@@ -0,0 +1,173 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: poppins, Helvetica, Arial, sans-serif;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+main {
```

Comment:
- [ ] Kindly adjust your stying for the main section, your page currently isn't responsive due to the fixed width and height, the design is supposed to fill the whole page in mobile view, but your design as shown below leaves some margins.

<img width="174" alt="image" src="https://user-images.githubusercontent.com/61408860/185468420-2dc9f823-8971-4d43-9534-e6a1c05a76ad.png">